process/Code review process.md

```diff
@@ -0,0 +1,131 @@
+# Code review process
+
+## Summary
+Description of the code review process used in log2timeline projects.
+
+## Rationale
+
+To keep the code base maintainable and readable all code is developed using a similar coding style. See the [style guide](https://github.com/log2timeline/plaso/wiki/Style-guide). This makes the code easier to maintain and understand.
+
+The purpose of the code review is to ensure that:
+
+ * at least two eyes looked over the code in hopes of finding potential bugs or errors (before they become bugs and errors). This also improves the overall code quality.
+ * make sure the code adheres to the style guide.
+ * review design decisions and if needed assist with making the code more optimal or error tolerant.
+
+**The short version:**
+
+*don't be intimidated.*
+
+**The longer version:**
+
+One language is not the same as another, you might be fluent in C or Perl, but that does not mean the same fluency for Python. You might have just started programming while others have been doing this for years. Our challenge is having a code base that is accessible and sufficiently uniform to most of you.
+
+Also don't be intimidated by rewrites/refactors, which can make it feel like the code base is changing under your feet. We have to make sure the code base is maintainable and this involves regular reshaping and cleaning up to get new features in.
+
+We continuously try to improve the code base, including making things and easier and quicker to write. This sometimes means that the way you just learned might already superseded by another. We try to keep the documentation up to date but this will sometimes be after you ran into an issue.
+
+First time contributors may come across the fact that the code review process can take quite a long time, with lots of back and forth comments. You may think that you are wasting the core developers time, but rest assured you are not. We look at this as an investment of building up good solid code contributors. We want to make sure our contributors understand the code and the style guide and will make suggestions to the contributor to fix what we think needs improving. Despite spending potentially more time to begin with to get code submitted into the project we believe this investment in code review will result in better code submissions and increased proficiency of the contributor.
+
+Therefore we would like to ask people to hang on, to get through the code review process and try to learn something while going through it. Rest assured, it will get easier next time and even easier the time after that, and before you know it you can contribute code to the project with little to no comments.
+
+And if things are unclear, don't hesitate to ask. The developer mailing list is: log2timeline-dev@googlegroups.com or you can join the slack channel [here](https://t.co/mO1re10Bfs). 
+
+## Process
```

Comment:
Maybe rename the heading to Detailed steps seeing the previous heading is already named Process

Reply:
Done